Client/Helpers/FxALoginHelper.swift

```diff
@@ -55,6 +55,11 @@ class FxALoginHelper {
             return loginDidFail()
         }
 
+        // If we've got here then we're loading an already logged in account.
+        // At this point, no one is interested if the account is verified or not,
+        // so assume it is.
+        accountVerified = true
```

Comment:
I completely missed the `.needsVerification` state which makes a lot more sense 🤘